examples/mixture/plot_gmm_selection.py

```diff
@@ -94,6 +94,7 @@
 
 plt.xticks(())
 plt.yticks(())
-plt.title('Selected GMM: full model, 2 components')
+plt.title('Selected GMM: {} model, {} components'.format(best_gmm.covariance_type,
+                                                         best_gmm.n_components))
```

Comment:
This will go above the 80 character line limit still. Suggestion:

```suggestion
plt.title('Selected GMM: {} model, {} components'.format(
    best_gmm.covariance_type, best_gmm.n_components))
```